buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -204,7 +204,7 @@
       "continueOnError": false,
       "alwaysRun": false,
       "displayName": "Build tests",
-      "condition": "and(succeeded(), ne(variables['PB_DisableTestBuild'], 'true'))",
+      "condition": "and(succeeded(), ne(variables.PB_SkipTests, 'true'))",
```

Comment:
Is there a difference between the `['...']` and `.` syntax (besides conciseness)?

Reply:
see variables under https://docs.microsoft.com/en-us/vsts/build-release/concepts/process/conditions. They are the same except the dot syntax has some restrictions on the naming of the variable but should impact us. I think it is a little more concise which is why I like it (one lets set of quotes to match up).